core-kotlin-modules/core-kotlin-concurrency-3/src/main/java/com/baeldung/comparecollectandcollectlatest/UsingCollect.kt

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.comparecollectandcollectlatest
+
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.runBlocking
+
+fun main() {
+    runBlocking {
+        flowOf(1, 2, 3)
+                .collect { value ->
+                    println(value)
+                }
+    }
```

Comment:
This formatting feels a bit off - would it make sense to arrange it like this instead?
The change is to put `.collect()` starting on the same line of flowOf(), so that we don't need quite so many indents.

This will also apply to the other snippet.

```kotlin
        flowOf(1, 2, 3).collect { value ->
            println(value)
        }
    }
```